### ISSUE TITLE
chapter3-Processing Raw Text

### ISSUE THREAD

**Issue by lvjiujin** [nltk/nltk_book#213]
url = "http://www.gutenberg.org/files/2554/2554.txt"
this url  is not found, so you can't access the url, if you access it, there will be a problem'404'
so I modified it to this url = "http://www.gutenberg.org/files/2554/2554-0.txt"

**Comment by HaelChan** [comment]
Yes, the original url is invalid. And the corresponding output is slightly different.

**Comment by Boundarybreaker** [comment]
`2554-0.txt` is now in UTF-8 already, so the `.decode('utf8')` command isn't needed anymore.
